application/dash_application/plot_builder.py

```diff
@@ -1,68 +1,108 @@
+#                 'triangle-down-open-dot',
+#                 'triangle-left-open', 'triangle-left-dot',
+#                 'triangle-left-open-dot',
+#                 'triangle-right-open', 'triangle-right-dot',
+#                 'triangle-right-open-dot', 'triangle-ne',
+#                 'triangle-ne-open', 'triangle-ne-dot',
+#                 'triangle-ne-open-dot', 'triangle-se',
+#                 'triangle-se-open', 'triangle-se-dot',
+#                 'triangle-se-open-dot', 'triangle-sw',
+#                 'triangle-sw-open', 'triangle-sw-dot',
+#                 'triangle-sw-open-dot', 'triangle-nw',
+#                 'triangle-nw-open', 'triangle-nw-dot',
+#                 'triangle-nw-open-dot', 'pentagon',
+#                 'pentagon-open', 'pentagon-dot',
+#                 'pentagon-open-dot', 'hexagon', 'hexagon-open',
+#                 'hexagon-dot', 'hexagon-open-dot',
+#                 'hexagon2', 'hexagon2-open', 'hexagon2-dot',
+#                 'hexagon2-open-dot', 'octagon',
+#                 'octagon-open', 'octagon-dot',
+#                 'octagon-open-dot', 'star',  'star-open',
+#                 'star-dot', 'star-open-dot', 'hexagram',
+#                 'hexagram-open', 'hexagram-dot',
+#                 'hexagram-open-dot', 'star-triangle-up',
+#                 'star-triangle-up-open', 'star-triangle-up-dot',
+#                 'star-triangle-up-open-dot', 'star-triangle-down',
+#                 'star-triangle-down-open',
+#                 'star-triangle-down-dot',
+#                 'star-triangle-down-open-dot', 'star-square',
+#                 'star-square-open','star-square-dot',
+#                 'star-square-open-dot', 'star-diamond',
+#                 'star-diamond-open', 'star-diamond-dot',
+#                 'star-diamond-open-dot', 'diamond-tall',
+#                 'diamond-tall-open', 'diamond-tall-dot',
+#                 'diamond-tall-open-dot', 'diamond-wide',
+#                 'diamond-wide-open', 'diamond-wide-dot',
+#                 'diamond-wide-open-dot', 'hourglass',
+#                 'hourglass-open', 'bowtie', 'bowtie-open',
+#                 'circle-cross', 'circle-cross-open', 'circle-x',
+#                 'circle-x-open', 'square-cross',
+#                 'square-cross-open', 'square-x', 'square-x-open',
+#                 'diamond-cross', 'diamond-cross-open',
+#                 'diamond-x', 'diamond-x-open', 'cross-thin',
+#                 'cross-thin-open', 'x-thin', 'x-thin-open',
+#                 'asterisk', 'asterisk-open', 'hash',
+#                 'hash-open', 'hash-dot', 'hash-open-dot',
+#                 'y-up', 'y-up-open', 'y-down',
+#                 'y-down-open', 'y-left', 'y-left-open',
+#                 'y-right', 'y-right-open', 'line-ew',
+#                 'line-ew-open', 'line-ns', 'line-ns-open',
+#                 'line-ne', 'line-ne-open', 'line-nw',
+#                 'line-nw-open']
+
+ALL_SYMBOLS = [
```

Comment:
this list got commented out then added back in?

Reply:
Yes -- sorry, tried to make that clearer in the commit message for this PR but should have added it to the file instead. Will update.